cmd/goal/wallet.go

```diff
@@ -166,7 +167,11 @@ var newWalletCmd = &cobra.Command{
 
 				// Display the mnemonic to the user
 				reportInfoln(infoPrintedBackupPhrase)
-				reportInfof(infoBackupPhrase, mnemonic)
+				if runtime.GOOS == "windows" {
+					reportInfof(infoBackupPhraseWindows, mnemonic)
+				} else {
+					reportInfof(infoBackupPhrase, mnemonic)
+				}
```

Comment:
instead of having the difference here, define two constants for `\x1B[32m` and `\033[0m` and use these.
The constants should be defined separately for windows and *nix.